levels.go

```diff
@@ -1000,6 +1000,7 @@ func (s *levelsController) getTableInfo(withKeysCount bool) (result []TableInfo)
 			var count uint64
 			if withKeysCount {
 				it := t.NewIterator(false)
+				defer it.Close()
```

Comment:
having `defer` in `for` loop is not good idea. Iterator can be closed after counting.

Reply:
Yeah. Thanks! Fixed it.